brjs-sdk/sdk/libs/javascript/br-i18n/src/br/i18n/Translator.js

```diff
@@ -386,6 +386,10 @@ Translator.prototype._getTranslationForKeyOrUndefined = function(token) {
 	token = token.toLowerCase();
 
 	if (!this.tokenExists(token)) {
+		if (typeof window.console === "undefined") {
```

Comment:
that won't fix the problem, the problem isn't that console.warn is undefined, its that window.console is undefined in ie8 when the console isnt open